docs/application/dotnet/guides/notification/attach-panel.md

```diff
@@ -68,6 +68,8 @@ To enable your application to use the attach panel functionality:
        <privilege>http://tizen.org/privilege/mediastorage</privilege>
        <!--To add the camera UI gadget in the attach panel-->
        <privilege>http://tizen.org/privilege/camera</privilege>
+       <!--To add the camera UI gadget in the attach panel-->
```

Comment:
Please can you change this sentence as:
<!--To add the telephony UI in the attach panel-->

Reply:
I re-paste sabeena21's comment here again.

```
Please can you change this sentence as:
<!--To add the telephony UI in the attach panel-->
```

@sabeena21 
If you use \< blah-blah \> form directly on here,
this editor will consider it as a kind of tag.

`<!--To add the telephony UI in the attach panel-->` is not a valid tag, 
so the editor of Github hide it.

Reply:
Okay @mijin85cho, sure I will make use of it.

Thank you for providing information about this.